controller/src/main/java/org/jboss/as/controller/ValidateModelStepHandler.java

```diff
@@ -113,5 +161,22 @@ boolean hasAlternative(final String[] alternatives, ModelNode operationObject) {
         return false;
     }
 
+    private Resource loadResource(OperationContext context) {
+        final PathAddress address = context.getCurrentAddress();
+        PathAddress current = PathAddress.EMPTY_ADDRESS;
+        Resource resource = context.readResourceFromRoot(PathAddress.EMPTY_ADDRESS, false);
+        for (PathElement element : address) {
```

Comment:
what is the point of this? try to find last resource that is still valid as part of path address?
What does that give us?